tests/helpers/resolver.js

```diff
@@ -1,9 +1,11 @@
-import Resolver from 'ember/resolver';
+import Resolver from 'ember-resolver';
+import config from '../../config/environment';
 
-var resolver = Resolver.create();
+const resolver = Resolver.create();
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).